src/server/simulator.js

```diff
@@ -51,7 +51,7 @@ function Simulator(opts) {
         updateDevice: this.updateDevice
     };
 
-    this._config.theme = theme.createTheme(simulatorProxy, opts.theme);
+    this.updateTheme(opts.theme);
```

Comment:
I am not exactly sure why we need to change it to updateTheme. What likely to happen is that we will put theme reloading logic in updateTheme() method later, so it completes the them refreshing mission. My interpretation of updateTheme means that "there is already an existing theme, and we are changing/updating it". From this semantic, using updateTheme() here might be over-assuming things in this context.